website_lazy_load_image/static/src/js/editor.js

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2018 Onestein
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+odoo.define('website_lazy_load_image.editor', function (require) {
+    'use strict';
+
+    var rte = require('web_editor.rte');
+    var RTEWidget = rte.Class;
+
+    RTEWidget.include({
+        _saveElement: function($el) {
+            this.prepareLazyImages($el.find('img'));
```

Comment:
I think we should take care also of background images

Reply:
:+1:  will do when I have time.